server/go/api_apis_service_impl.go

```diff
@@ -35,14 +35,14 @@ func NewApisApiService(kuskClient kusk.Client) ApisApiServicer {
 
 // GetApis - Get a list of APIs
 func (s *ApisApiService) GetApis(ctx context.Context, namespace string, fleetname string, fleetnamespace string) (ImplResponse, error) {
-	apis := &kuskv1.APIList{}
+	var apis *kuskv1.APIList
 	var err error
+
 	if fleetname == "" {
 		apis, err = s.kuskClient.GetApis(namespace)
 		if err != nil {
 			return Response(http.StatusInternalServerError, err), err
 		}
-
 	} else {
 		apis, err = s.kuskClient.GetApiByEnvoyFleet(namespace, fleetnamespace, fleetname)
```

Comment:
```suggestion
		apis, err := s.kuskClient.GetApiByEnvoyFleet(namespace, fleetnamespace, fleetname)
```

Reply:
same as above I need to return apis at the end of the function